### PR TITLE
Feat: Added Card Components

### PR DESCRIPTION
**Pull request by shuntzhere**
Added Multiple variants of cards:

- Vertical card
- Text Only
- Text Overlay
- Badge Card
- Dismissable card
(I will be adding more and Ignore the HTML file)

https://user-images.githubusercontent.com/47948956/153436943-f220e033-a33d-4a51-8c4b-105e59a99700.mp4

### CODE REVIEW

**Review by shubhamkumar648 — COMMENTED** [review]
you forgot to add alt attribute in img tag of card html file ! remaing looks good

**Comment by shuntzhere** [comment]
> you forgot to add alt attribute in img tag of card html file ! remaing looks good

Ah updated that. Thanks for reviewing.

**Comment by shubhamkumar648** [comment]
> > you forgot to add alt attribute in img tag of card html file ! remaing looks good
> 
> Ah updated that. Thanks for reviewing.

perfect

**Review by shubhamkumar648 — APPROVED** [review]

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
+.card {
+  height: 450px;
+  width: 350px;
+  background-color: white;
```
Comment (Sameony) [inline components/card/card.css:4]: You have a white colored variable so i reckon you can use that here

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  padding-top: 0.5rem;
+  font-size: 2rem;
+  color: #474747;
+}
+
+.card-text {
+  display: inline-block;
+  margin: 0.2rem auto;
+  font-size: 1rem;
+  color: #474747;
```
Comment (Sameony) [inline components/card/card.css:27]: You can create variables to maintain consistency. Same for color in line no. 31

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  font-size: 1.5rem;
+  overflow: hidden;
+}
+
+span {
+  display: inline-block;
+  font-size: 1.5rem;
+}
+
+/* textonly card  */
```
Comment (Sameony) [inline components/card/card.css:41]: Since the class name is descriptive enough, you can remove the comment here

Comment (Sameony) [reply components/card/card.css:41]: Same for line no. 48 and 76

File: components/card/card.html [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+      <div class="nav-container">
+        <nav>
+          <div class="left-nav flex flex-items-center justify-between">
+            <img class="hamburger" src="/assets/svg/hamburger-solid.svg" />
+            <a href="./index.html">
+              <img class="logo flex-grow" src="/assets/svg/logo.svg" />
+            </a>
+            <a class="right-nav flex" href="#">Docs</a>
+          </div>
+          <!-- <div></div> -->
```
Comment (Sameony) [inline components/card/card.html:21]: Remove unnecessary comments and realign and refactor the code

**Review by Sameony — COMMENTED** [review]
Some minute adjustments. Aside from that, good code, nice variable names, very descriptive and unambiguous

File: components/card/card.css (hunk 1 above)
Comment (shuntzhere) [reply components/card/card.css:4]: Yaah, so I've updated this will be there by next push

File: components/card/card.css (hunk 3 above)
Comment (shuntzhere) [reply components/card/card.css:41]: Sure will change that while refactoring.

File: components/card/card.css (hunk 2 above)
Comment (shuntzhere) [reply components/card/card.css:27]: It's updated now

**Review by Sameony — APPROVED** [review]

**Pull request merged**